src/dotnet/CarbonAware.Tools.WattTimeClient/Configuration/ServiceCollectionExtensions.cs

```diff
@@ -50,4 +55,14 @@ public static IServiceCollection ConfigureWattTimeClient(this IServiceCollection
 
         return services;
     }
+
+    private static void LogProxyConfiguration(IConfiguration config, CarbonAwareVariablesConfiguration caVars)
+    {
+        ILoggerFactory factory = LoggerFactory.Create(b => {
+            b.AddConfiguration(config.GetSection("Logging"));
+            b.AddConsole();
+        });
+        var logger = factory.CreateLogger<IServiceCollection>();
```

Comment:
I am not sure if we should log the username. If we really wish to log it for debugging purposes, maybe we log a hash of username.

Reply:
No preference here. I think we are only doing this for troubleshooting purposes, so hashing the username would require that the consumer/client knows what the hash is about.

Reply:
Makes sense. I still dont think it is advisable to log username, but I am fine with whatever the team thinks is right. Will resolve the conversation.